docs/conf.py

```diff
@@ -25,7 +25,13 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.viewcode']
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.doctest',
+    'sphinx.ext.viewcode',
+    'sphinx.ext.intersphinx',
+    'repoze.sphinx.autointerface',
```

Comment:
@jamadden This change caused `tox -e docs` to break, as it was not installing `repoze.sphinx.autointerface`.  I fixed that in 357914bf7b15081caff5970711ac043cce10bb2c.